python/ray/worker.py

```diff
@@ -2454,10 +2450,11 @@ def decorator(function_or_class):
 
             # Set the actor default resources.
             if num_cpus is None and num_gpus is None and resources is None:
-                # In the default case, actors acquire no resources for
-                # their lifetime, and actor methods will require 1 CPU.
-                cpus_to_use = DEFAULT_ACTOR_CREATION_CPUS_SIMPLE_CASE
-                actor_method_cpus = DEFAULT_ACTOR_METHOD_CPUS_SIMPLE_CASE
+                # If the resources are not specified in the decorator, we wait
```

Comment:
It seems like we should be able to avoid all of the logic here and just pass the values `num_cpus`, `num_gpus`, and `resources` through to `make_actor`, right? Do we actually need any of this logic here? It should be determined when the actor is instantiated.